src/content/article/over-a-year-late-a-dollar-short-more-than-two-decades-of-kodi/index.md

```diff
@@ -0,0 +1,87 @@
+
+The Xbox changed all that. In 2001, Microsoft released their first console, leveraging their work on x86 platforms to build out a version of Windows specialized for running games. The familiarity allowed game developers to easily port to the new gaming console; the fact that Microsoft sold them at a loss, knowing they would make it back in game sales, ensured rapid consumer adoption. Luckily for the hackers out there, it was more similar to than different from something the majority were familiar with: the standard personal computer running Windows. Very quickly, folks like Bunnie figured out ways around the security, achieving the holy grail in record time for a console: the ability to run untrusted code.
+
+Long before we had the Raspberry Pi, OpenWrt routers and other tinkerer's dream "open" environments, Xbox hacking quickly became the fastest route to general-purpose, connected computing. The availability of powerful, affordable, standardized hardware enabled an entire community of hackers wanting to break out of Microsoft’s walled garden and run whatever code they wanted on their Xbox. Emulation, or the ability to trick old games such as NES and Sega Genesis into believing the Xbox is the same hardware, was one of the first big use cases, since we now had something powerful enough that hooked up to a regular TV and offered a standard controller.
+
+**XBMC is Born**
+
+While many folk worked on getting Linux and various other things running, the project that attracted me the most was Xbox Media Player. This brought the ability to play compressed media to the Xbox: up until this point, only a computer could play these files, and if you were one of the those super early adopters who had network cables strung across your house, you could now even stream it over the network or the Internet (which was definitely not fast enough for video… yet!).
+
+[XBMP](https://sourceforge.net/projects/xbplayer/) came out in 2002 as the convergence of two separate closed source apps natively written with the Microsoft Xbox Development Kit by **d7o3g4q** (a.k.a. duo) and **RUNTiME**. As many at the time pointed out, since it was leveraging GPL code, it likely needed to be open source. At the same time, another developer - **Frodo** - had created [YAMP](https://sourceforge.net/projects/xbyamp/) as ‘Yet Another Media Player’; when this project also converged at the end of 2002,[ Xbox Media Center](https://sourceforge.net/projects/xbmc/), or XBMC, was born. While the initial beta was closed source, the developers quickly open sourced it and it’s been open source ever since.
```

Comment:
```suggestion
[XBMP](https://sourceforge.net/projects/xbplayer/) came out in 2002 as the convergence of two separate closed source apps natively written with the Microsoft Xbox Development Kit by **d7o3g4q** (a.k.a. duo) and **RUNTiME**. As many at the time pointed out, since it was leveraging GPL code, it likely needed to be open source. At the same time, another developer - **Frodo** - had created [YAMP](https://sourceforge.net/projects/xbyamp/) as ‘Yet Another Media Player’; when this project also converged at the end of 2002, [Xbox Media Center](https://sourceforge.net/projects/xbmc/), or XBMC, was born. While the initial beta was closed source, the developers quickly open sourced it and it’s been open source ever since.
```

Reply:
Done